tests/test_cache.py

```diff
@@ -57,6 +57,19 @@ def test_get_cache_key(self):
         key = cache_backend.get_cache_key(SampleResourceModel, url)
         assert key == "note::http://foo.com/v1/expected_title/"
 
```

Comment:
This test is exactly the same as the test `test_get_cache_key` except that it has query parameters passed to the lookup and therefore in the url.